### DISCUSSION TITLE
Compatibility with 11.3 ?

### DISCUSSION THREAD

**Discussion by Erwan-Shom** [Esri/arcgis-powershell-dsc#540] [Q&A] [answered]
Hi,
great tool !
is arcgis-powershell-dsc compatible with ArcGIS Enterprise 11.3 / ArcGIS Pro 3.0 release ?
Thanks

**Comment by cameronkroeker** [comment, marked as answer]
Hi @Erwan-Shom,

Yes, with the latest release, v4.3.0 it supports ArcGIS Enterprise 11.3 and ArcGIS Pro 3.3:

https://github.com/Esri/arcgis-powershell-dsc/releases/tag/v4.3.0

Thanks,
Cameron K.
